runtime/jcl/common/vm_scar.c

```diff
@@ -278,7 +278,7 @@ scarInit(J9JavaVM * vm)
 #if defined(WIN32) && !defined(OPENJ9_BUILD)
 	switch (jclVersion) {
 	case J2SE_18:
-	case J2SE_19:	/* This will need to be modified when the JCL team provides a dbgwrapper90 */
+	case J2SE_V11:	/* This will need to be modified when the JCL team provides a dbgwrapper90 */
```

Comment:
Java 11 doesn't support dbgwrapper and probably never will. It was only used for an IBM internal Java 9 that was obsoleted by OpenJ9.

Reply:
Will remove `J2SE_V11` as well then.

Reply:
PR has been updated, please have another look.